tidb-cloud/api-overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TiDB Cloud API Overview
+summary: Learn about what is TiDB Cloud API, its features, and how to use API to manage your TiDB Cloud clusters.
+---
+
+# TiDB Cloud API Overview <span style="color: #fff; background-color: #00bfff; border-radius: 4px; font-size: 0.5em; vertical-align: middle; margin-left: 16px; padding: 0 2px;">Beta</span>
```

Comment:
Added `beta` tag as requested by PM 
<img width="423" alt="image" src="https://user-images.githubusercontent.com/59654584/185830974-e8d041ea-e50f-4952-93de-f766f2e9046e.png">

Reply:
a preview: https://preview-docs.netlify.app/zh/tidb/dev/api-overview

Reply:
Will the PDF render this title as expected?

Reply:
I'm not sure. I can't generate pdf locally.